configure.ac

```
@@ -1776,18 +1776,22 @@ AC_ARG_ENABLE([experimental-jit],
               [],
               [enable_experimental_jit=no])
 case $enable_experimental_jit in
-  no) enable_experimental_jit=no ;;
-  yes) enable_experimental_jit="-D_Py_JIT -D_Py_TIER2=1" ;;
-  yes-off) enable_experimental_jit="-D_Py_JIT -D_Py_TIER2=3" ;;
-  interpreter) enable_experimental_jit="-D_Py_TIER2=4" ;;
-  interpreter-off) enable_experimental_jit="-D_Py_TIER2=6" ;;  # Secret option
+  no)              jit_flags="";          tier2_flags="" ;;
+  yes)             jit_flags="-D_Py_JIT"; tier2_flags="-D_Py_TIER2=1" ;;
+  yes-off)         jit_flags="-D_Py_JIT"; tier2_flags="-D_Py_TIER2=3" ;;
+  interpreter)     jit_flags="";          tier2_flags="-D_Py_TIER2=4" ;;
+  interpreter-off) jit_flags="";          tier2_flags="-D_Py_TIER2=6" ;;  # Secret option
   *) AC_MSG_ERROR(
       [invalid argument: --enable-experimental-jit=$enable_experimental_jit; expected no|yes|yes-off|interpreter]) ;;
 esac
-AS_VAR_IF([enable_experimental_jit],
-          [no],
+AS_VAR_IF([tier2_flags],
           [],
-          [AS_VAR_APPEND([CFLAGS_NODIST], [" $enable_experimental_jit"])
+          [],
+          [AS_VAR_APPEND([CFLAGS_NODIST], [" $tier2_flags"])])
+AS_VAR_IF([jit_flags],
+          [],
+          [],
+          [AS_VAR_APPEND([CFLAGS_NODIST], [" $jit_flags"])
```

Sorry for the late review; looks like the `AS_VAR_IF` on L1791 was not closed, nor is the opening quote on L1794 closed. I'd expect this:

```suggestion
          [AS_VAR_APPEND([CFLAGS_NODIST], [" $jit_flags"])])
```

---

Ah, I see what's done now; it seems to be correctly closed in L1801. Sorry for the noise!

---

Blame it on the Lisp-y convention for closing brackets. :-)